alternator/serialization.hh

```diff
@@ -12,7 +12,7 @@
 #include <string_view>
 #include <optional>
 #include "types.hh"
-#include "schema_fwd.hh"
+#include "schema_basic_types.hh"
```

Comment:
I think this patch is just noise. Nobody complained that schema_fwd.hh wasn't 1000% accurate, everyone understood what it meant. Now they have to learn something new for no gain.

Reply:
Botond thought that adding table_id and table_schema_version to it are out of scope.
Therefore the rename serves to increase the scope of the header file.
I don't mind keeping it as is and add the definitions there.

We can define them just in schema.hh but the occasional users will have to include schema.hh that has a lot of bagage.  Maybe that's ok.

However, I do mind defining it the same way in two places, e.g both in schema.hh and schema_fwd.hh.
That's a very bad habbit.

Reply:
Doesn't schema.hh include schema_fwd.hh?

Reply:
It definitely should have, but it didn't up until this series.